integration/common/setup.py

```diff
@@ -23,6 +23,7 @@
 
 requirements = [
     "attrs>=19.3.0",
+    "Jinja2",
```

Comment:
We'll be using this only for dbt and will get this as a transitive dependency there - no need to add it to requirements.